src/python/pants/util/docutil.py

```diff
@@ -16,3 +16,13 @@ def terminal_width(*, fallback: int = 96, padding: int = 2) -> int:
 
 def doc_url(slug: str) -> str:
     return f"https://www.pantsbuild.org/v{MAJOR_MINOR}/docs/{slug}"
+
+
+def git_url(fp: str) -> str:
+    """Link to code in pantsbuild/pants.
+
+    Note that links will not be stable for dev releases because `main` floats. Only release
+    candidates and stable releases have stable links.
+    """
+    branch = "main" if PANTS_SEMVER.is_devrelease else f"{MAJOR_MINOR}.x"
```

Comment:
Even this floats when we cherrypick, we should probably use the release tag?

Reply:
> Even this floats when we cherrypick

What do you mean? This value only matters when we run `generate_docs.py` and when the user runs `pantsbuild.pants`. We run `generate_docs.py` from the release branch, so we should be good there. The `pantsbuild.pants` will have embedded the correct VERSION.

Reply:
What about console help?

Reply:
Oh I see, nm.

Reply:
Actually, my point stands. The references will be to the HEAD of 2.7.x, say, which may be different in 2.7.0 vs 2.7.1. The patch version matters, no?

Reply:
Hm, yeah, true. Any ideas on how else to approach this? I don't think we can reliably get the git SHA so that we can link to the stable link.

It's too error-prone imo to have this type of drift of 2.7.0 vs 2.7.1 - I'd be concerned someone trying to debug an install will errantly think the 2.7.1 lockfile is being used when really it's 2.7.0, and they're led astray.

I'd rather not embed the default into `./pants help`, but maybe that's the best we can do?

Reply:
I think we can use the release tag in the git URL?

Reply:
Ah, yes! Brilliant, thanks Benjy :)